src/examples/rest/src/backend/schema/task/entity.ts

```diff
@@ -34,6 +35,43 @@ const acl: AccessControlList<Task, AuthorizationContext> = {
 	},
 };
 
+registerEnumType(Priority, {
+	name: 'Priority',
+	valuesConfig: {
+		HIGH: {
+			description: 'HIGH',
+		},
+		MEDIUM: {
+			description: 'MEDIUM',
+		},
+		LOW: {
+			description: 'LOW',
+		},
+	},
+});
+
+type TaskField = keyof InstanceType<typeof Task>;
+
+export const preventLightSideAccess = (
```

Comment:
Maybe just add a longer comment of what is going on here about how Luke is part of the Light side and is unable to see the priority column etc, etc